{{ cookiecutter.repo_name }}/docker-compose.override.yml

```diff
@@ -0,0 +1,21 @@
+version: '2.1'
+services:
+    dask-worker:
+      image: drtools/{{ cookiecutter.repo_name }}:dev
+      volumes:
+        - ${MODULES_PATH-..}/sparsity:/home/drtools/sparsity
```

Comment:
I'm not an expert on docker-compose but I think you can't reference out of the parent directory so  `..` might not work.